src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1862,6 +1861,22 @@ MavlinkReceiver::handle_message_rc_channels_override(mavlink_message_t *msg)
 	rc.values[16] = man.chan17_raw;
 	rc.values[17] = man.chan18_raw;
 
+	// check how many channels are valid
+	for (int i = 17; i >= 0; i--) {
```

Comment:
I still don't care for the magic number, but I don't see anything from mavlink we can use directly.